WordPress/Classes/ViewRelated/Blog/Blog + Me/UIBarButtonItem+MeBarButton.swift

```diff
@@ -1,39 +1,26 @@
 import Gridicons
 import UIKit
 
-
 /// Add a UIBarButtonItem to the navigation bar that  presents the Me scene.
-extension BlogDetailsViewController {
-    @objc
-    func presentHandler() {
-        meScenePresenter.present(on: self, animated: true, completion: nil)
-    }
-
+extension UIViewController {
```

Comment:
totally agree with removing the repeated method; one minor thing is that we are implicitly saying that any `UIViewController` can add a Me button. We could use a protocol and then conform only the viewControllers we want, but then again we would need to implement those methods (unless we use a default implementation, which is less then ideal anyway)

Reply:
I kinda tried this approach but if felt a bit like an "unnecessary" extra step, since we'd be providing a default implementation of this method, and I don't think the VC would ever override that implementation.

That's not to say I feel too strongly about this.  I'd be game to implement the protocol if that feels cleaner to you.

Thoughts?

Reply:
I think it's fine to leave as it is for now. Also, it's pretty clear what the method does so it should not cause any problem.